api/src/lib/aws.ts

```diff
@@ -143,8 +143,7 @@ function getSQSClient() {
   return sqs
 }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This function's signature doesn't appear to have any unused vars, so removed this pragma.